### PR TITLE
Removed Python 2 Files

### PR DESCRIPTION
**Pull request by lancemathias**
Removed Python 2 and Python 2 libraries, and Python 2 Matplotlib since Python 2 is end of life.

### CODE REVIEW

File: modules/ocf/manifests/extrapackages.pp [hunk 1]
```diff
@@ -15,7 +15,6 @@
   include ocf::packages::chrome
   include ocf::packages::emacs
   include ocf::packages::kubectl
-  include ocf::packages::matplotlib
```
Comment (ethanhs) [inline modules/ocf/manifests/extrapackages.pp:18]: I don't think this should be removed, otherwise looks great!

Comment (lancemathias) [reply modules/ocf/manifests/extrapackages.pp:18]: Should be fixed!

**Review by ethanhs — APPROVED** [review]
If something actually relies on Python 2, god help us.

**Comment by singingtelegram** [comment]
ipython is also based on Python 2 and no longer in the Debian 11 package repositories.

**Pull request merged**